src/main.js

```diff
@@ -1,7 +1,12 @@
+/**
```

Comment:
This is more of a project description and less useful as a file-level comment imo